upsert/upsert.go

```diff
@@ -0,0 +1,46 @@
+package upsert
+
+import (
+	"errors"
+	"net/http"
+	"strings"
+)
+
+var (
+	//ErrInvalidResponse is returned when the Search API returns a non 200 status
```

Comment:
Please put a space at the start of the comment
```suggestion
	// ErrInvalidResponse is returned when the Search API returns a non 200 status
```